core/services/cable_guy/main.py

```diff
@@ -137,3 +137,4 @@ def trigger_dynamic_ip_acquisition(interface_name: str) -> Any:
     server = Server(config)
 
     loop.run_until_complete(server.serve())
+    manager.stop()
```

Comment:
This should be not necessary since it will be called by __del__.

Reply:
Agree.